src/platform/silabs/wifi/rs911x/WifiInterfaceImpl.cpp

```diff
@@ -72,12 +74,48 @@ osThreadAttr_t kDrvTaskAttr = { .name       = "drv_rsi",
                                 .stack_size = kDrvTaskSize,
                                 .priority   = osPriorityHigh };
 
-static osMessageQueueId_t sWifiEventQueue = NULL;
+osMessageQueueId_t sWifiEventQueue = NULL;
+
+uint8_t wfx_rsi_drv_buf[WFX_RSI_BUF_SZ];
+wfx_wifi_scan_ext_t temp_reset;
+
+osTimerId_t sDHCPTimer;
+
+} // namespace
+
 /*
  * This file implements the interface to the RSI SAPIs
  */
-static uint8_t wfx_rsi_drv_buf[WFX_RSI_BUF_SZ];
-static wfx_wifi_scan_ext_t temp_reset;
+
+void DHCPTimerEventHandler(void * arg)
```

Comment:
Since these functions are removed from the headers then port over the comments from the header file to this file to have a description on the same

Reply:
Added the required comments